api-module-library/hubspot/tests/api.test.js

```diff
@@ -299,36 +299,50 @@ describe(`${config.label} API tests`, () => {
             primaryLandingPages = await api.getLandingPages('translatedFromId__is_null');
             expect(primaryLandingPages).toBeDefined();
         });
-        let variationLandingPages
+        let variationLandingPages;
+        let sampleLandingPage;
         it('should return only variation language landing pages', async () => {
             variationLandingPages = await api.getLandingPages('translatedFromId__not_null');
             expect(variationLandingPages).toBeDefined();
+            sampleLandingPage = variationLandingPages.results.slice(-1)[0];
+            expect(sampleLandingPage.id).toBeDefined();
         });
         it('confirm total landing pages', async () => {
             expect(allLandingPages.total).toBe(primaryLandingPages.total + variationLandingPages.total)
         });
+
         it('get Landing Page by Id' , async () => {
-            const pageToGet = variationLandingPages.results.slice(-1)[0];
-            const response = await api.getLandingPage(pageToGet.id);
+            const response = await api.getLandingPage(sampleLandingPage.id);
             expect(response).toBeDefined();
         });
-        it('update a Landing page (minimal patch)' , async () => {
-            const pageToUpdate = variationLandingPages.results.slice(-1)[0];
+        it('update a Landing page (maximal patch)' , async () => {
+            delete sampleLandingPage['archivedAt'];
             const response = await api.updateLandingPage(
-                pageToUpdate.id,
-                {htmlTitle: `test Landing page ${Date.now()}`},
+                sampleLandingPage.id,
+                sampleLandingPage,
                 true );
             expect(response).toBeDefined();
         });
-        it('update a Landing page (maximal patch)' , async () => {
-            const pageToUpdate = variationLandingPages.results.slice(-1)[0];
-            delete pageToUpdate['archivedAt'];
+        it('update a Landing page (minimal patch)' , async () => {
             const response = await api.updateLandingPage(
-                pageToUpdate.id,
-                pageToUpdate,
+                sampleLandingPage.id,
+                {htmlTitle: `test Landing page ${Date.now()}`},
                 true );
             expect(response).toBeDefined();
         });
```

Comment:
I like this approach, although it might not make sense to always define expect output in the arrange block, from my view.

That said I think it would be out of scope to change all the tests to be like that for this PR